pkg/proc/core/core.go

```diff
@@ -222,9 +220,13 @@ func OpenCore(corePath, exePath string, debugInfoDirs []string) (*proc.Target, e
 // initialize for core files doesn't do much
 // aside from call the post initialization setup.
 func (p *Process) initialize(path string, debugInfoDirs []string) error {
-	return proc.PostInitializationSetup(p, path, debugInfoDirs, p.writeBreakpoint)
+	return proc.PostInitializationSetup(p, path, debugInfoDirs, p.WriteBreakpointFn)
```

Comment:
Since WriteBreakpointFn is now part of the Process interface there is no reason to pass it here.

Reply:
Done.